### PR TITLE
support scroll/resize end/start

### PR DESCRIPTION
**Pull request by hankhsiao**
Support scrollStart, scrollEnd, resizeStart, and resizeEnd.

### CODE REVIEW

**Comment by hankhsiao** [comment]
@roderickhsiao do you want to review :dancer:

**Comment by roderickhsiao** [comment]
:shipit:

File: package.json [hunk 1]
```diff
@@ -51,8 +51,8 @@
     "mocha": "^2.0",
     "mockery": "^1.4.0",
     "pre-commit": "^1.0.0",
-    "react": "<= 0.13.x",
     "react-tools": "<= 0.13.x",
```
Comment (roderickhsiao) [inline package.json]: `react-tools` is deprecated

File: src/leIE8.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ * Copyright 2015, Yahoo! Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+/* global navigator, parseFloat */
+'use strict';
+
+var leIE8 = false; // less then or equal to IE8
+
+if (typeof navigator !== 'undefined') {
+    var matches = navigator.userAgent.match(/MSIE (\d+\.\d+)/);
```
Comment (roderickhsiao) [inline src/leIE8.js]: probably regex in variable?

File: src/leIE8.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/**
```
Comment (roderickhsiao) [inline src/leIE8.js]: I am thinking can we change this to be more generic, e.g. 
`eligibleBrowser` , and validate a set of browser blacklist (IE 8 in this case)

File: package.json [hunk 4]
```diff
@@ -43,6 +42,7 @@
     "grunt-saucelabs": "^8.3.2",
     "grunt-shell": "^1.1.2",
     "grunt-webpack": "^1.0.8",
+    "grunt": "^0.4.5",
     "istanbul": "^0.3.0",
```
Comment (roderickhsiao) [inline package.json]: `jenkins-mocha` ?

File: package.json [hunk 5]
```diff
@@ -51,8 +51,8 @@
     "mocha": "^2.0",
     "mockery": "^1.4.0",
     "pre-commit": "^1.0.0",
-    "react": "<= 0.13.x",
     "react-tools": "<= 0.13.x",
+    "react": "<= 0.13.x",
     "xunit-file": "^0.0.5"
```
Comment (roderickhsiao) [inline package.json]: we probably don't need this

**Pull request merged**